plugins/inputs/nvidia_smi/nvidia_smi.go

```diff
@@ -31,20 +31,31 @@ func (smi *NvidiaSMI) Description() string {
 // SampleConfig returns the sample configuration for the NvidiaSMI plugin
 func (smi *NvidiaSMI) SampleConfig() string {
 	return `
-  ## Optional: path to nvidia-smi binary, defaults to $PATH via exec.LookPath
+  ## Optional: path to nvidia-smi binary, defaults "/usr/bin/nvidia-smi"
+  ## We will first try to locate the nvidia-smi binary with the explicitly specified value (or default value), 
+  ## if it is not found, we will try to locate it on PATH(exec.LookPath), if it is still not found, an error will be returned
   # bin_path = "/usr/bin/nvidia-smi"
 
-  ## Optional: timeout for GPU polling
+  ## Optional: timeout for GPU polling, defaults "5s"
```

Comment:
You don't need to add the `, defaults "5s"` here. The convention is that the default value is the commented value.

Reply:
ok